tests/shared/core/test_domain.py

```diff
@@ -722,11 +722,11 @@ def test_domain_from_multiple_files():
     ]
 
     expected_entities = [
+        "pistachio_robot",
```

Comment:
So... the previous version was sorted alphabetically because the list of entities returned was sorted.
I wonder if it's possible to keep this list in alphabetical order, and then do something similar to what we do with `domain_slots` below:
```
domain_entities = []
for entity in domain.entities:
        domain_slots.append(entity.name)
```

and then ` assert expected_entities == sorted(domain_entities)`

Reply:
I just sorted in the assert statement since the entities are already just a list of names